secp256k1_test.go

```diff
@@ -0,0 +1,534 @@
+package secp256k1
+
+import (
+	"bytes"
+	"encoding/hex"
+	"math/big"
+	"math/rand"
+	"testing"
+)
+
+// TODO: Add test vectors. (even though they're included in libsecp itself)
+
+const LoopsN = 150
```

Comment:
Also, does LoopsN need to be exported? If you aren't using it anywhere outside of this package, please unexport it.

Reply:
It's a test variable for how many times to run the loops in the tests. it's used both in secp256k1_test and in multiset_test, I can put different ones in each file though

Reply:
Unlike Rust, files don't define scope. Meaning that since both files are inside the package "secp256k1" if you unexport LoopsN here it will still be available in multiset_test.go.